src/ontology/mondo-edit.obo

```diff
@@ -438962,6 +438965,74 @@ is_a: MONDO:0100355 {source="https://orcid.org/0000-0001-5208-3432"} ! classic o
 property_value: seeAlso https://github.com/monarch-initiative/mondo/issues/3221 xsd:string
 created_by: http://orcid.org/0000-0001-5208-3432
 
+[Term]
+id: MONDO:0100358
+name: lip herpes
+def: "Any herpes simplex infection that involves the lip." [MONDO:patterns/location]
+synonym: "herpes infectious disease of lip" EXACT [MONDO:patterns/location]
+synonym: "lip herpes infectious disease" EXACT [MONDO:patterns/location]
+is_a: MONDO:0004609 {source="https://orcid.org/0000-0001-5208-3432"} ! herpes simplex infectious disease
+intersection_of: MONDO:0004609 ! herpes simplex infectious disease
+intersection_of: disease_has_location UBERON:0001833 ! lip
+property_value: seeAlso https://github.com/monarch-initiative/mondo/issues/3348 xsd:string
+created_by: http://orcid.org/0000-0001-5208-3432
+
+[Term]
+id: MONDO:0100359
+name: herpes simplex type 1 infectious disease
+def: "A disease caused by infection with herpes simplex type 1." [MONDO:patterns/infectious_disease_by_agent]
+is_a: MONDO:0004609 {source="https://orcid.org/0000-0001-5208-3432"} ! herpes simplex infectious disease
+property_value: seeAlso https://github.com/monarch-initiative/mondo/issues/3348 xsd:string
+created_by: http://orcid.org/0000-0001-5208-3432
+
```

Comment:
can we add the EquivalentTo "herpes simplex infectious disease' and ('disease has infectious agent' 'herpes simplex type 1')?
(we could argue about whether it should be 'reactivation of latent agent' or not)

Reply:
Yes, I intend to do that. I am having trouble with the NCBI taxon import, I am going to ask Nico about that on the Mondo call today.

Reply:
done